.github/mergify.yml

```
@@ -2,0 +1,552 @@
+      - -merged
+      - -closed
+
+  # Was recently updated one minute ago
+  is_recent_commit: &is_recent_commit
+    - and:
+      - commits[*].date_committer > 1 minutes ago
+
+  # README.md is updated together with README.yaml
+  readme_updated: &readme_updated
+    - or:
+      - and:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)